8Knot/queries/query_template.py

```diff
@@ -1,22 +1,20 @@
 import logging
 import pandas as pd
-from db_manager.augur_manager import AugurManager
-from app import celery_app
-from cache_manager.cache_manager import CacheManager as cm
+import cache_manager.cache_facade as cf
 import io
-import datetime as dt
-from sqlalchemy.exc import SQLAlchemyError
 
 """
 TODO:
 (1) update QUERY_NAME
 (2) update 'NAME_query' found in function definition and in the function call that sets the 'ack' variable below.
 'NAME' should be the same as QUERY_NAME
-(3) paste SQL query in the query_string
+(3) paste SQL query in the query_string. Make sure it's wild-carded for psycopg2
 (4) insert any necessary df column name or format changed under the pandas column and format updates comment
 (5) reset df index if #4 is performed via "df = df.reset_index(drop=True)"
 (6) go to index/index_callbacks.py and import the NAME_query as a unqiue acronym and add it to the QUERIES list
-(7) delete this list when completed
+(8) specify the correct table name in the call to 'caching_wrapper'
+(9) create a table in 8Knot/8Knot/cache_manager/db_init.py for the new data you're requesting.
+(10) delete this list when completed
```

Comment:
if preprocessing_utils should be where any pandas preprocessing should be add to the list explaining what that is and how it should be used. Then any comments of pandas preprocessing of queries should be removed from the query files and moved over there for people to refer to

Reply:
++